sympy/solvers/diophantine/diophantine.py

```diff
@@ -140,24 +140,20 @@ class DiophantineEquationType:
     """
     Internal representation of a particular diophantine equation type.
 
-    Parameters
+    Attributes
     ==========
 
-    equation
+    equation :
         The diophantine equation that is being solved.
-    free_symbols: list (optional)
+    free_symbols : list (optional)
         The symbols being solved for.
```

Comment:
Apologies, I might not have made it very clear. `equation` and `free_symbols` are parameters and everything else are attributes.